it/controllers/benefits/accommodation/LivingAccommodationBenefitsControllerISpec.scala

```diff
@@ -32,24 +37,15 @@ import utils.{EmploymentDatabaseHelper, IntegrationTest, ViewHelpers}
 
 class LivingAccommodationBenefitsControllerISpec extends IntegrationTest with ViewHelpers with EmploymentDatabaseHelper {
 
-  val taxYearEOY: Int = 2021
-  val employmentId: String = "001"
+  private val taxYearEOY: Int = 2021
+  private val employmentId: String = "employmentId"
```

Comment:
are these employmentId variables needed? or can you just reference DefaultUser.employmentId? this would be in multiple test specs

Reply:
fair enough